### PR TITLE
feat: Nodejs8.10 is EOL 

### PR DESCRIPTION
**Pull request by hideokamoto**
# Title
AWS Lambda (Nodejs8.10) will be EOL at the end of the year.

## Description
AWS Lambda version should be greater than nodejs10.x.
Because the Nodejs8.10 will be EOL at the end of the year.

|Name | Identifier | End of Life | Deprecation (Create) | Deprecation (Update)|
|-- | -- | -- | -- | --|
|Node.js 0.10 | nodejs | October 31, 2016 | October 31, 2016 | October 31, 2016|
|Node.js 4.3 | nodejs4.3nodejs4.3-edge | April 30, 2018 | December 15, 2018 | April 30, 2019|
|Node.js 6.10 | nodejs6.10 | April 30, 2019 | April 30, 2019 | August 12, 2019|
|Node.js 8.10 | nodejs8.10 | December 31, 2019 | January 6, 2020 | February 3, 2020|

from: https://docs.aws.amazon.com/lambda/latest/dg/runtime-support-policy.html

So we have to upgrade our AWS Lambda version.

## Types of changes

- [ x ] Bug fix (non-breaking change which fixes an issue)
- [ ] New feature (non-breaking change which adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to change)

## Checklist

- [ ] My code follows the code style of this project
- [ ] My change requires a change to the documentation
- [ ] I have updated the documentation accordingly
- [ ] I have read the **README** document
- [ ] I have added tests to cover my changes
- [ ] All new and existing tests passed

## License

- [x] I confirm that this pull request can be released under the Apache 2 license

[issues]: https://github.com/alexa-games/litexa/issues
[license]: https://www.apache.org/licenses/LICENSE-2.0
[cla]: http://en.wikipedia.org/wiki/Contributor_License_Agreement

### CODE REVIEW

**Comment by ayrc** [comment]
Hello Hidetaka,

Thank you for your pull request! A few things:

Can you please change the target branch to develop? We have a contribution guideline [here](https://github.com/alexa-games/litexa/blob/master/CONTRIBUTING.md) for more details.

Second, can you update `docs/book/appendix-default-aws-settings.md` to include the Node.js version in the Lambda section? You can use `npm run docs:dev` to preview the documentation locally.

Thank you!
Alina

**Comment by hideokamoto** [comment]
I've update the target.
But when I runed the `npm run docs:dev` command in my local, I got the error.

```bash
> litexa@ docs:dev /Users/develop/alexa/litexa
> npx vuepress dev docs


 WAIT  Extracting site metadata...
Error: Cannot find module '/usr/local/opt/nvm/versions/node/v10.5.0/lib/node_modules/markdown-it-vuepress-code-snippet-enhanced'
    at Function.Module._resolveFilename (internal/modules/cjs/loader.js:636:15)
    at Function.Module._load (internal/modules/cjs/loader.js:562:25)
    at Module.require (internal/modules/cjs/loader.js:690:17)
    at require (internal/modules/cjs/helpers.js:25:18)
    at Object.config (/Users/develop/alexa/litexa/docs/.vuepress/config.js:31:16)
    at module.exports (/Users/develop/alexa/litexa/node_modules/vuepress/lib/markdown/index.js:52:14)
    at resolveOptions (/Users/develop/alexa/litexa/node_modules/vuepress/lib/prepare/resolveOptions.js:106:20)
    at prepare (/Users/develop/alexa/litexa/node_modules/vuepress/lib/prepare/index.js:11:25)
    at dev (/Users/develop/alexa/litexa/node_modules/vuepress/lib/dev.js:23:25)
    at args (/Users/develop/alexa/litexa/node_modules/vuepress/bin/vuepress.js:105:12)
npm ERR! code ELIFECYCLE
npm ERR! errno 1
npm ERR! litexa@ docs:dev: `npx vuepress dev docs`
npm ERR! Exit status 1
npm ERR! 
npm ERR! Failed at the litexa@ docs:dev script.
npm ERR! This is probably not a problem with npm. There is likely additional logging output above.

npm ERR! A complete log of this run can be found in:
npm ERR!     /Users/dc_hideokamoto/.npm/_logs/2019-12-23T14_01_09_436Z-debug.log
```

## ENV

```
$ node -v
v10.16.0

$ npm -v
6.9.0

$ git log -2
commit e760980406a464d78337a22f32876bc7fa45d9e6 (HEAD -> feat/aws/lambda-version, origin/feat/aws/lambda-version)
Author: hideokamoto <hide.okamoto@digitalcube.jp>
Date:   Sun Dec 15 02:45:41 2019 +0900

    feat: upgrade aws lambda runtime version

commit edb292d970396a070f8cd8938be9c813491d1082 (origin/master, origin/develop, origin/HEAD, master)
Merge: 15f0278 4e001d7
Author: Johannes Seifert <48674557+joseifert@users.noreply.github.com>
Date:   Thu Oct 24 14:31:51 2019 -0700

    Merge pull request #74 from alexa-games/develop
    
    chore: merge for release


```

**Comment by joseifert** [comment]
Hi @hideokamoto. To build and preview the docs locally, you'll need to follow the steps in the README (https://github.com/alexa-games/litexa#one-time-setup) and install the following 2 NPM packages globally:
- `npm install -g vuepress`
- `npm install -g markdown-it-vuepress-code-snippet-enhanced`

**Comment by hideokamoto** [comment]
I've update the doc today!
The CLI use `node12.x` but the `node10.x` is not EOL soon.
So the docs say `Use node10.x or later`.

And I've added the document about the EOL.
https://docs.aws.amazon.com/lambda/latest/dg/runtime-support-policy.html

Thanks.

File: docs/book/appendix-default-aws-settings.md [hunk 1]
```diff
@@ -17,6 +17,8 @@ gives this the skill requests's `context.System.device.deviceId` field, by defau
 
 ## Lambda
 
+* Lambda runtime should use `nodejs10.x` or later.
+  * We can check the runtime EOL in [the AWS document](https://docs.aws.amazon.com/lambda/latest/dg/runtime-support-policy.html).
```
Comment (ayrc) [inline docs/book/appendix-default-aws-settings.md]: The content is good. Can you reword the sentence to be more specific and consistent with the rest of the docs? For example, we generally try to say 'you' when referring to an action the user can take. You can rewrite it to be `You can check the runtime deprecation schedule in the [Lambda Runtime Support Policy](...)` or something similar. Thank you!

**Comment by ayrc** [comment]
Merged commit now; will update documentation in a followup PR.

**Comment by hideokamoto** [comment]
Thanks!!

**Pull request merged**